Stable

kidalabama opened the pull request:
db_search.php i want select all tables without click select all button default.

[http://oi66.tinypic.com/206e7n8.jpg](http://oi66.tinypic.com/206e7n8.jpg)

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [x] Any new functionality is covered by tests

ibennetch committed:
Merge branch 'QA_4_5' into STABLE

ibennetch committed:
Merge branch 'MAINT_4_5_5' into STABLE

ibennetch committed:
Merge branch 'QA_3_6' into STABLE

ibennetch committed:
Merge branch 'QA_4_6' into STABLE

ibennetch committed:
Merge branch 'QA_4_6' into STABLE

ibennetch committed:
Merge branch 'QA_4_6' into STABLE

ibennetch committed:
Merge branch 'QA_4_6' into STABLE

ibennetch committed:
Merge branch 'QA_4_6' into STABLE

ibennetch committed:
Merge branch 'QA_4_6' into STABLE

ibennetch committed:
Merge branch 'QA_4_6' into STABLE

ibennetch committed:
Merge branch 'QA_4_6' into STABLE

ibennetch committed:
Merge branch 'MAINT_4_6_5' into STABLE

ibennetch committed:
Merge branch 'QA_4_6' into STABLE

ibennetch committed:
Merge branch 'QA_4_7' into STABLE

ibennetch committed:
Merge branch 'QA_4_7' into STABLE

ibennetch committed:
Merge branch 'QA_4_7' into STABLE

ibennetch committed:
Merge branch 'QA_4_7' into STABLE

ibennetch committed:
Merge branch 'QA_4_7' into STABLE

nijel committed:
Translated using Weblate (English (United Kingdom))

Currently translated at 100.0% (3205 of 3205 strings)

[CI skip]

Seitanas committed:
Fix for password change issues on Galera cluster

Signed-off-by: Tadas Ustinavičius <tadas@ring.lt>

ibennetch committed:
Merge branch 'QA_4_7' into STABLE

ibennetch committed:
Prepare for 4.7.6-dev

Signed-off-by: Isaac Bennetch <bennetch@gmail.com>

gnbezerra committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (3205 of 3205 strings)

[CI skip]

nijel committed:
Remove composer.lock after release

Signed-off-by: Michal Čihař <michal@cihar.com>

stevenwdv committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (3205 of 3205 strings)

[CI skip]

nijel committed:
Ignore php-myadmin.ru from linkcheck in docs

The site works, it's just missing intermediate certificate, what makes
it fail in the linkcheck, but usually works in the browser as it has
already seen the intermediate cert before elsewhere.

Signed-off-by: Michal Čihař <michal@cihar.com>

MauricioFauth committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (3205 of 3205 strings)

[CI skip]

Commit:
Translated using Weblate (Finnish)

Currently translated at 69.0% (2212 of 3205 strings)

[CI skip]

nijel committed:
Fixed check all interaction with filtering

- check all select only filtered rows
- the hidden rows are automatically unselected
- the check all state is updated according to filtering

Fixes #13517

Signed-off-by: Michal Čihař <michal@cihar.com>

nijel committed:
Workaround SNI problems on readthedocs.org

Signed-off-by: Michal Čihař <michal@cihar.com>

Commit:
Translated using Weblate (Thai)

Currently translated at 28.8% (926 of 3205 strings)

[CI skip]

nijel committed:
Clarify comment and add another host to ignore list

Signed-off-by: Michal Čihař <michal@cihar.com>

nijel committed:
Update po files

[CI skip]

Signed-off-by: Michal Čihař <michal@cihar.com>

Commit:
Translated using Weblate (Interlingua)

Currently translated at 75.5% (2423 of 3207 strings)

[CI skip]

BouRock committed:
Translated using Weblate (Turkish)

Currently translated at 100.0% (3205 of 3205 strings)

[CI skip]

victorenator committed:
Translated using Weblate (Belarusian)

Currently translated at 63.3% (2032 of 3207 strings)

[CI skip]

BouRock committed:
Translated using Weblate (Turkish)

Currently translated at 100.0% (3205 of 3205 strings)

[CI skip]

nijel committed:
Optimize PNG images with zopfli

See #13797

Signed-off-by: Michal Čihař <michal@cihar.com>

madhuracj committed:
Fix #13800 Database lost when renaming to similar name and lower_case_table_names=1

Signed-off-by: Madhura Jayaratne <madhura.cj@gmail.com>

Commit:
Translated using Weblate (Estonian)

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

stefanomarty committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

dingo1313 committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

dingo1313 committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

marsjaninzmarsa committed:
Translated using Weblate (Polish)

Currently translated at 81.2% (2606 of 3207 strings)

[CI skip]

marsjaninzmarsa committed:
Translated using Weblate (Polish)

Currently translated at 81.2% (2607 of 3207 strings)

[CI skip]

Commit:
Translated using Weblate (German)

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

Commit:
Translated using Weblate (German)

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

readmanr committed:
Translated using Weblate (English (United Kingdom))

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

vgoncalves21 committed:
Translated using Weblate (Portuguese)

Currently translated at 55.4% (1778 of 3207 strings)

[CI skip]

Commit:
Translated using Weblate (Slovenian)

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

jsmakaayb committed:
Translated using Weblate (Danish)

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

nijel committed:
Add SJIS-win to default list of allowed charsets

Fixes #13803

Signed-off-by: Michal Čihař <michal@cihar.com>

Commit:
Translated using Weblate (English (United Kingdom))

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

MauricioFauth committed:
Translated using Weblate (English (United Kingdom))

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

MauricioFauth committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

Commit:
Translated using Weblate (Kyrgyz)

Currently translated at 1.8% (59 of 3207 strings)

[CI skip]

nijel committed:
Translated using Weblate (Greek)

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

nijel committed:
Translated using Weblate (French)

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

nijel committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

nijel committed:
Translated using Weblate (Indonesian)

Currently translated at 82.2% (2637 of 3207 strings)

[CI skip]

nijel committed:
Translated using Weblate (Greek)

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

nijel committed:
Translated using Weblate (Russian)

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

nijel committed:
Translated using Weblate (Albanian)

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

nijel committed:
Fix typo in transformation

Signed-off-by: Michal Čihař <michal@cihar.com>

Commit:
Translated using Weblate (Estonian)

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

Cymrodor committed:
Translated using Weblate (Welsh)

Currently translated at 16.2% (522 of 3207 strings)

[CI skip]

nijel committed:
Translated using Weblate (Welsh)

Currently translated at 16.2% (522 of 3207 strings)

[CI skip]

tmtisfree committed:
Translated using Weblate (French)

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

nijel committed:
Detect another way MySQL server needs SSL

Fixes #13436

Signed-off-by: Michal Čihař <michal@cihar.com>

shazaum committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

AironPereira committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

dingo1313 committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

Commit:
Translated using Weblate (Estonian)

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

msbastos committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 99.6% (3197 of 3207 strings)

[CI skip]

aref-lazrek committed:
Translated using Weblate (Arabic)

Currently translated at 49.3% (1583 of 3207 strings)

[CI skip]

Commit:
Translated using Weblate (Hungarian)

Currently translated at 98.4% (3157 of 3207 strings)

[CI skip]

lordark616 committed:
Translated using Weblate (Polish)

Currently translated at 81.2% (2952 of 3207 strings)

[CI skip]

lordark616 committed:
Translated using Weblate (Polish)

Currently translated at 98.5% (3160 of 3207 strings)

[CI skip]

nijel committed:
Use correct name for motranslator call

We're not using php gettext, so using _ is mistake.

Issue #13436

Signed-off-by: Michal Čihař <michal@cihar.com>

nijel committed:
Correctly unset session on logout

Signed-off-by: Michal Čihař <michal@cihar.com>

nijel committed:
Add support for JSON field in MariaDB

Fixs #13038

Signed-off-by: Michal Čihař <michal@cihar.com>

nijel committed:
Avoid using empty and workarounding its match to zero

Signed-off-by: Michal Čihař <michal@cihar.com>

nijel committed:
Fixed constructing ALTER query with AFTER

We have to include only non empty fields in this case.

Fixes #13824

Signed-off-by: Michal Čihař <michal@cihar.com>

Commit:
Translated using Weblate (French)

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

Commit:
Translated using Weblate (English (United Kingdom))

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

nijel committed:
Gracefully fail on missing parameter

Fixes #13841

Signed-off-by: Michal Čihař <michal@cihar.com>

mregazzi committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

nijel committed:
Lock page when changes are done in the SQL editor

The CodeMirror does not update underlying textarea in all cases, so we
need to handle it directly.

Fixes #13821

Signed-off-by: Michal Čihař <michal@cihar.com>

nijel committed:
Revert "Translated using Weblate (English (United Kingdom))"

This reverts commit 663f1f3138b488c78a5ed95b2e0d945429f2e59c.

nijel committed:
Prefer iconv for encoding conversions

The mb extension is usually present, however it does not support that
wide range of charsers.

Basically this reverts d846b2e5d59fc442b77f3868cddbce9cd3172fd7.

Fixes #13842

Signed-off-by: Michal Čihař <michal@cihar.com>

nijel committed:
Fix coding style for #13737

nijel committed:
Set $sql_query as well when changing password

Issue #13737

Signed-off-by: Michal Čihař <michal@cihar.com>

nijel committed:
Changelog entry for #13737

Signed-off-by: Michal Čihař <michal@cihar.com>

Commit:
Translated using Weblate (Hungarian)

Currently translated at 98.8% (3170 of 3207 strings)

[CI skip]

ibennetch committed:
Documentation for ShowColumnComments

Signed-off-by: Isaac Bennetch <bennetch@gmail.com>

Commit:
Translated using Weblate (Dutch)

Currently translated at 100.0% (3207 of 3207 strings)

[CI skip]

nijel committed:
Compatibility with phpseclib 2.0.8

Apparently upgrade from 2.0.7 to 2.0.8 changed API in several ways.

See https://github.com/phpseclib/phpseclib/issues/1218

Fixes #13846

Signed-off-by: Michal Čihař <michal@cihar.com>

nijel committed:
Revert "Compatibility with phpseclib 2.0.8"

This reverts commit a450935e53a130319b84ce180a3456009e2c04b4.

We really need to stick on 2.0.7 for 4.7 branch to keep PHP 5.5 support.

Issue #13846

nijel committed:
Stick with phpseclib < 2.0.8 in 4.7 branch

We can not remove PHP 5.5 support from maintenance branch, so there is
no other way than sticking with older phpseclib.

See https://github.com/phpseclib/phpseclib/issues/1218

Issue #13846

Signed-off-by: Michal Čihař <michal@cihar.com>

Commit:
Translated using Weblate (Interlingua)

Currently translated at 75.5% (2424 of 3207 strings)

[CI skip]

nijel committed:
Revert limitation on phpseclib dependency

Only 2.0.8 is broken, so conflict with that.

Fixes #13846

Signed-off-by: Michal Čihař <michal@cihar.com>

ibennetch committed:
Prepare for 4.7.6 release

Signed-off-by: Isaac Bennetch <bennetch@gmail.com>

ibennetch committed:
Adding composer lock for 4.7.6

Signed-off-by: Isaac Bennetch <bennetch@gmail.com>

ibennetch committed:
Remove composer.lock after release

Signed-off-by: Isaac Bennetch <bennetch@gmail.com>

ibennetch committed:
Merge branch 'QA_4_7' into STABLE

ibennetch committed:
Merge branch 'QA_4_7' into STABLE

ibennetch committed:
Merge branch 'QA_4_7' into STABLE

ibennetch committed:
Merge branch 'QA_4_8' into STABLE

ibennetch committed:
Merge branch 'MAINT_4_8_0' into STABLE

ibennetch committed:
Merge branch 'QA_4_8' into STABLE

ibennetch committed:
Merge branch 'QA_4_8' into STABLE

williamdes committed:
Fix getDatabasesFull query

Execute: $this->dbi->getDatabasesFull("pma_test")
Fixes missing space after alias

Signed-off-by: William Desportes <williamdes@wdes.fr>

aswaniprakash committed:
drop table issue on copying tables fixed #14426

Signed-off-by: Aswani Prakash <aswani15prakash@gmail.com>
(cherry picked from commit 8fc4d862a81e1e3b19a3ad5686aaef33726892d7)
Signed-off-by: Maurício Meneghini Fauth <mauriciofauth@gmail.com>

williamdes committed:
Fixes #14422 - table comment not showing

Closes: #14422
Signed-off-by: William Desportes <williamdes@wdes.fr>

adlerweb committed:
Translated using Weblate (German)

Currently translated at 100.0% (3288 of 3288 strings)

[CI skip]

Translation: phpMyAdmin/4.8
Translate-URL: https://hosted.weblate.org/projects/phpmyadmin/4-8/de/
Signed-off-by: Florian Knodt <git@adlerweb.info>

popcorner committed:
Translated using Weblate (Chinese (Traditional))

Currently translated at 94.7% (3114 of 3288 strings)

[CI skip]

Translation: phpMyAdmin/4.8
Translate-URL: https://hosted.weblate.org/projects/phpmyadmin/4-8/zh_Hant/
Signed-off-by: popcorner <memoword@163.com>

dancaragea committed:
Translated using Weblate (Romanian)

Currently translated at 96.6% (3177 of 3288 strings)

[CI skip]

Translation: phpMyAdmin/4.8
Translate-URL: https://hosted.weblate.org/projects/phpmyadmin/4-8/ro/
Signed-off-by: Dan Caragea <dan_caragea@yahoo.com>

Commit:
Translated using Weblate (German)

Currently translated at 100.0% (3288 of 3288 strings)

[CI skip]

Translation: phpMyAdmin/4.8
Translate-URL: https://hosted.weblate.org/projects/phpmyadmin/4-8/de/
Signed-off-by: BO41 <lukasw41@gmail.com>

ibennetch commented:
Hi @kidalabama I'm a little confused by this. You seem to have opened a pull request, which is usually meant for submitting some programming contribution.

I think you meant to open a feature request instead, but I'm not sure what you're requesting. Do you think the search page should load with the tables selected by default? This seems like a reasonable suggestion to me.

MauricioFauth committed:
Merge pull request #14550 from williamdes/issue-14422-table-comment-bug

Fixes #14422 - table comment not showing

weblate committed:
Merge branch 'origin/QA_4_8' into Weblate

MauricioFauth committed:
Add ChangeLog entry for #14422

Signed-off-by: Maurício Meneghini Fauth <mauriciofauth@gmail.com>

MauricioFauth committed:
Add ChangeLog entry for #14426

Signed-off-by: Maurício Meneghini Fauth <mauriciofauth@gmail.com>

jaideejung007 committed:
Translated using Weblate (Thai)

Currently translated at 29.6% (976 of 3288 strings)

[CI skip]

Translation: phpMyAdmin/4.8
Translate-URL: https://hosted.weblate.org/projects/phpmyadmin/4-8/th/
Signed-off-by: Anusuk Sangubon <jaideejung007@gmail.com>

Commit:
Translated using Weblate (Estonian)

Currently translated at 100.0% (3288 of 3288 strings)

[CI skip]

Translation: phpMyAdmin/4.8
Translate-URL: https://hosted.weblate.org/projects/phpmyadmin/4-8/et/
Signed-off-by: Kristjan Räts <kristjanrats@gmail.com>

kidalabama commented:
yes "the search page should load with the tables selected by default"!!!

madhuracj committed:
Fix #14548 [security] HTML injection in import warning messages

Signed-off-by: Madhura Jayaratne <madhura.cj@gmail.com>

VictorAlagwu committed:
Fix escaped HTML in 'Add a new server' setup (#14585)

Fixes #14581

Signed-off-by: Victor Alagwu <victoralagwu@gmail.com>
(cherry picked from commit ec4331b7ce4f805e9cbd76fae074953284884c9f)
Signed-off-by: Maurício Meneghini Fauth <mauriciofauth@gmail.com>

MauricioFauth committed:
Add ChangeLog entry for #14581

Signed-off-by: Maurício Meneghini Fauth <mauriciofauth@gmail.com>

ibennetch committed:
Fix merge conflicts in ChangeLog

Signed-off-by: Isaac Bennetch <bennetch@gmail.com>

ibennetch committed:
Merge branch 'madhuracj-bug14548' into QA_4_8

ibennetch committed:
Prepare for version 4.8.3

Signed-off-by: Isaac Bennetch <bennetch@gmail.com>

ibennetch committed:
Add PMASA to ChangeLog

Signed-off-by: Isaac Bennetch <bennetch@gmail.com>

ibennetch committed:
Adding composer lock for 4.8.3

Signed-off-by: Isaac Bennetch <bennetch@gmail.com>

ibennetch committed:
Removing composer.lock

Signed-off-by: Isaac Bennetch <bennetch@gmail.com>

ibennetch committed:
Merge branch 'QA_4_8' into STABLE

ibennetch commented:
Moved to #14594.

You may wish to go there and click the "Subscribe" button to follow updates there.

Thanks for the suggestion.
